src/otx/core/model/entity/classification.py

```diff
@@ -673,7 +709,7 @@ def _customize_outputs(
             # Squeeze dim 4D => 3D, (1, num_classes, H, W) => (num_classes, H, W)
             predicted_s_maps = [out.saliency_map[0] for out in outputs]
 
-            predicted_f_vectors = [out.feature_vector for out in outputs]
+            predicted_f_vectors = [out.feature_vector[0] for out in outputs]
```

Comment:
Would you add a comment here about the reason why we extract index = 0 from `out.feature_vector`?

Reply:
Sure, added